zb-property.js

```diff
@@ -10,6 +10,7 @@
 
 'use strict';
 
+const Color = require('color');
```

Comment:
You'll need to add this dependency to `package.json`, and it has to be packaged with the add-on. @hobinjk can tell you how he did that for the Hue adapter.